Connected System Template (CST) Examples/Google Drive (OAuth,  Sending Files and Multiple Operations)/src/main/java/com/mycorp/helloworld/templates/GoogleJsonResponseExceptionHandler.java

```diff
@@ -0,0 +1,35 @@
+package com.mycorp.helloworld.templates;
+
+import com.appiancorp.connectedsystems.templateframework.sdk.IntegrationError;
+import com.appiancorp.connectedsystems.templateframework.sdk.IntegrationResponse;
+import com.appiancorp.connectedsystems.templateframework.sdk.diagnostics.IntegrationDesignerDiagnostic;
+import com.appiancorp.connectedsystems.templateframework.sdk.oauth.ExpiredTokenException;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+
+public class GoogleJsonResponseExceptionHandler {
+  public static IntegrationResponse handleException(
```

Comment:
I think this is okay but could probably be done inline as a method. More classes can make these examples harder to digest

Reply:
Talked in person. 
Putting this in a template class will result in redundant code.